.github/workflows/ci.yml

```diff
@@ -20,19 +20,14 @@ jobs:
       - name: free-up-space
         run: |
           df -h
-          sudo rm -rf /usr/share/dotnet   
+          sudo rm -rf /usr/share/dotnet
           sudo rm -rf /opt/ghc
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
           df -h
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      # -- REMOVE ONCE OPEN SOURCE
-      # Needed until Qiskit/qss-qasm is public.
-      - uses: webfactory/ssh-agent@v0.7.0
-        with:
-          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
You may need to restore this section if you want to git clone dependencies via SSH.

Reply:
Actually, it would be easier just to clone `qe-qasm` via HTTPS.